packages/matplotlib-inline/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: matplotlib-inline
+  version: 0.1.6
+  top-level:
+    - matplotlib_inline
+source:
+  url: https://files.pythonhosted.org/packages/f2/51/c34d7a1d528efaae3d8ddb18ef45a41f284eacf9e514523b191b7d0872cc/matplotlib_inline-0.1.6-py3-none-any.whl
+  sha256: f1f41aab5328aa5aaea9b16d083b128102f8712542f819fe7e6a420ff581b311
+about:
+  home: https://github.com/ipython/matplotlib-inline
+  PyPI: https://pypi.org/project/matplotlib-inline
+  summary: Inline Matplotlib backend for Jupyter
+  license: BSD 3-Clause
+extra:
+  recipe-maintainers:
+    - PUT_YOUR_GITHUB_USERNAME_HERE
```

Comment:
@mkoeppe I guess there should be your name (Or you remove the extra field completely, like in ipython/meta.yaml)